sophus/so3.hpp

```diff
@@ -336,7 +336,7 @@ class SO3Base {
       imag_factor = Scalar(0.5) - Scalar(1.0 / 48.0) * theta_sq +
                     Scalar(1.0 / 3840.0) * theta_po4;
       real_factor =
-          Scalar(1) - Scalar(0.5) * theta_sq + Scalar(1.0 / 384.0) * theta_po4;
+          Scalar(1) - Scalar(1.0 / 8.0) * theta_sq + Scalar(1.0 / 384.0) * theta_po4;
```

Comment:
Good catch: https://www.wolframalpha.com/input/?i=cos(t%2F2)

I wonder whether we'll be able to find add a unit test which would fail given the current/worng approximation, but pass using the correct one (e.g. by tightening the threshold maybe).